notebooks/samples/explanations/ai-explanations-tabnet-algorithm.ipynb

```diff
@@ -0,0 +1,540 @@
+{
```

Comment:
<p>Can you remove "The lighter color indicates more important feature." from the section title?</p><p>Instead, please add a sentence or two explaining how to interpret this plot.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/182/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>